bin/techsupport

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+TMPDIR=./techsupport
+mkdir -p $TMPDIR
+
+echo Collecting system information
+sudo ovs-vsctl show > $TMPDIR/ovs_vsctl_show.txt
+lsb_release -a > $TMPDIR/lsb_release.txt 2>/dev/null
+ip route > $TMPDIR/ip_route.txt
+ip link show > $TMPDIR/ip_link_show.txt
+arp -n > $TMPDIR/arp.txt
+
+if [ -d inst ]; then
+    echo Collecting run logs and config from 'inst' and 'local'
+    cp -r inst $TMPDIR
+    cp -r local $TMPDIR
+fi
+
+if [ -f local/system.conf ]; then
+    source local/system.conf
+fi
+
+# If an external switch was set up, try pinging it and saving that capture
+if [ -n "$ext_ctrl" ] && [ -n "$ext_addr" ]; then
+    echo Saving packet capture of ping to external switch
+    sudo timeout 3s tcpdump -i $ext_ctrl -w $TMPDIR/ping_switch.pcap &>/dev/null &
```

Comment:
might increase to 4s to avoid race conditions on startup -- rather capture a bit more than too little

Reply:
addressed.